classes/VideoService.php

```diff
@@ -461,6 +461,8 @@ class VideoService {
 			'https_enabled'	=> true,
 			'url_regex'		=> [
 				'#v=([\d\w-]+)(?:&\S+?)?#is',
+				'#youtube\.com/embed/([\d\w-]+)#is',
+				'#youtube\.com/shorts/([\d\w-]+)#is',
```

Comment:
Just a minor suggestion to reduce redundancy and make rules more manageable. Not a blocker.

```suggestion
				'#youtube\.com/(?:embed|shorts)/([\d\w-]+)#is',
```